tests/dummy/app/styles/app.scss

```diff
@@ -6,7 +6,6 @@
  */
 
 body {
-  overflow: hidden;
```

Comment:
I don't know why I needed to do this but if I didn't remove this I wasn't able to scroll the dummy app